shoop/core/models/_order_lines.py

```diff
@@ -118,7 +118,9 @@ def save(self, *args, **kwargs):
         if self.product_id and not self.supplier_id:
             raise ValidationError("Order line has product but no supplier")
 
-        return super(OrderLine, self).save(*args, **kwargs)
+        super(OrderLine, self).save(*args, **kwargs)
+        if self.product_id:
+            self.supplier.module.update_stock(self.product_id)
```

Comment:
Yes. As far as I know models save does not have to return anything. And if I remember correctly the return option was not even used anywhere in our code.